docker-compose.yml

```diff
@@ -80,7 +80,7 @@ services:
     command: >
       /bin/bash -c '
       sleep 3;
-      celery -A bootcamp worker -B -l debug'
+      celery -A bootcamp worker -B -l ${BOOTCAMP_LOG_LEVEL:-INFO}'
```

Comment:
Unless `version` at the top is changed to `2.1`, I get this error on OSX when running `docker-compose build`:

`Invalid interpolation format for "command" option in service "celery": "/bin/bash -c ' sleep 3; celery -A bootcamp worker -B -l ${BOOTCAMP_LOG_LEVEL:-INFO}'"`

Reply:
I changed it to 2.1